src/shared/compatibility.js

```diff
@@ -131,7 +131,7 @@ PDFJS.compatibilityChecked = true;
   if (String.prototype.includes) {
     return;
   }
-  String.prototype.includes = require('core-js/fn/string/includes');
```

Comment:
Just remove the `String.prototype.includes =` part and this should work.

When you import `core-js/fn/string/includes`, then `String.prototype.includes` is already overwritten, which can be seen using the following example:

```
$ node -e 'delete String.prototype.includes; console.log("".includes); require("core-js/fn/string/includes"); console.log("abc".includes("a"))'
undefined
true
```

The exported function from `require('core-js/fn/string/includes')` does not use `this`, but expects the first parameter to be the string, and the other parameters to be the parameters for the string method. I.e. `require('core-js/fn/string/includes')("abc", "b");`. With the current (wrong) code, it becomes require('core-js/fn/string/includes').call("abc", "b")`, which becomes (since `this` is ignored) `require('core-js/fn/string/includes')("b")`, i.e. equivalent to `"b".includes(undefined)`.

The documentation at https://github.com/zloirock/core-js#usage also shows that the intended usage for the default mode is to just call `require` and not use its return value.

Reply:
Thank you; I've made the suggested changes and everything seems to work just fine now!